resolwe_bio/docker_images/rnaseq/Dockerfile

```diff
@@ -1,3 +1,6 @@
 FROM docker.io/resolwebio/common:1.0.0
 
 MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bio
+
+# Force matplotlib backend to a headless one.
+ENV MPLBACKEND Agg
```

Comment:
I suggest we add this to the common image where the matplotlib is installed.

Reply:
Why not? Add it to common and we'll build and publish common:1.0.1 which we can reference here.

Reply:
you can do both in the same PR

Reply:
Good point, will do.